R/example.R

```diff
@@ -0,0 +1,43 @@
+pieEx1 = list(
```

Comment:
This does not look like a good example in my eyes. Ideally we should not require users to prepare a huge list for `echart()`, otherwise that defeats the purpose of this package. Instead, we want something as simple as `echart(data, ~variable, type = 'pie')`. Do you have a better example?